azure/hub/variables.tf

```diff
@@ -97,6 +97,7 @@ variable "firewall_config" {
     subnet_prefix      = string
     public_ip_name     = optional(string)
     sku_tier           = optional(string, "Standard")
+    zones              = optional(list(string), [1,2,3])
```

Comment:
This default value is not required here since it matches the default value set in the firewall module. Leaving this as simply `optional(list(string))` would suffice?

Reply:
changed this to zone_redundant